tests/utils/index.ts

```diff
@@ -297,3 +297,15 @@ export const openTestPage = async (page: Page, url: string, queryMap?: Record<st
 
     await goto(page, fullUrl, {isRetry: false});
 };
+
+export const fillDatePicker = async (args: {page: Page; selector: string; value?: string}) => {
+    const {page, selector, value = ''} = args;
+    const datePicker = await page.waitForSelector(selector);
```

Comment:
waitForSelector is [deprecated](waitForSelector)
it should be converted to locators